GitUI/UserControls/FileStatusList.cs

```diff
@@ -712,11 +759,30 @@ public void StoreNextIndexToSelect()
             _nextIndexToSelect = _nextIndexToSelect - FileStatusListView.SelectedIndices.Count + 1;
         }
 
+        public void SortByFilePath()
+        {
+            FileStatusListView.ListViewItemSorter = new GitStatusListSorter(Comparer<GitItemStatus>.Default);
+            FileStatusListView.Sort();
+        }
+
+        public void SortByFileExtension()
+        {
+            FileStatusListView.ListViewItemSorter = new GitStatusListSorter(new GitItemStatusFileExtensionComparer());
+            FileStatusListView.Sort();
+        }
+
+        public void SortByFileStatus()
+        {
+            FileStatusListView.ListViewItemSorter = new ImageIndexListSorter();
+            FileStatusListView.Sort();
+        }
```

Comment:
These new methods don't need to be `public`.
We can make them public when there is a real need to use them from outside.

Reply:
Will do. That was careless of me. At one point I was calling them externally but didn't clean that up when I refactored. Thanks for catching that.

Reply:
If you want to test these (great that you do!), please follow the `TestAccessor` pattern we have